### DISCUSSION TITLE
Visualization of individual pairs

### DISCUSSION THREAD

**Discussion by jpulecio** [Katsevich-Lab/sceptre#156] [Q&A] [answered]
Is it possible to visualize the results of individual pairs after running the sceptre at-scale code? We are trying to use an alternative approach by fetching the data using the CellRanger output and Seurat to process gene expression. This works to extract the expression values for the gRNA_Cells but we are not able to properly fetch the information from the Control_Cells.

**Comment by ekatsevi** [comment]
Please see Discussion #148.

**Reply by jpulecio** [reply]
Thanks for the answer!  We actually tried to run the Nextflow pipeline to plot the individual response_grna_target_pairs. We are able to get the sceptre_object and then after that we re-ran the assign_grnas and runqc to get a new sceptre_object that allows us to run function [plot_response_grna_target_pair].

However, when we run assign_grnas on the sceptre_object, we got the error below:
sceptre_object <- assign_grnas(
sceptre_object = sceptre_object,
formula_object = formula_object,
method = "mixture"
)
Error in set_matrix_accessibility(grna_matrix, make_row_accessible = TRUE) :
Class not recognized.

Do you have an idea on how to fix this?

**Reply by ekatsevi** [reply, marked as answer]
We are looking into this issue, which other users have also encountered. Our current theory for the origin of this error is that the `sceptre_object` returned by the Nextflow pipeline is not meant to be operated on in the same way as regular `sceptre_objects`, because certain fields are removed by the pipeline for memory efficiency purposes. You might have to rerun `assign_grnas()` and `run_qc()` on a newly created `sceptre_object` instead of the one output by the pipeline. Could you please try this?
